jdaviz/configs/imviz/plugins/orientation/orientation.vue

```diff
@@ -14,6 +14,10 @@
           Please add at least one data with valid WCS to align by sky (WCS).
         </v-alert>
 
+        <v-alert v-if="wcs_linking_available" type='warning' style="margin-left: -12px; margin-right: -12px">
+          Switching orientation will reset zoom.

```

Comment:
is it switching orientation or switching between pixel and WCS linking?  If the latter - can we have similar language here and for the markers/subsets warnings?

Reply:
I think both. The point is switching reference data resets zoom. I don't think glue cares which one.

Reply:
Actually, wait, maybe Brett did a special handling on one but not the other. Let me check and get back to you. Thanks!

Reply:
OK just the link type, not when you switch between different WCS orientations. I updated the wording. Looks good now? Good catch!